src/components/PathwayCourse/index.js

```diff
@@ -395,29 +397,28 @@ function PathwayCourse() {
                     container
                     spacing={2}
                     align="center"
-                    className={classes.box}
-                  >
+                    className={classes.box}>
                     <Grid item xs={12} md={6} sx={{ pl: 2 }}>
                       <Card
                         align="left"
                         elevation={0}
                         className={classes.titleCard}
-                        mb={isActive ? 16 : 30}
-                      >
+                        mb={isActive ? 16 : 30}>
                         {pathwayCourseData.code !== "SHCEL" && (
                           <Typography
                             variant="body2"
                             className={classes.cardSubtitle}
-                            sx={{ textAlign: isActive && "center", pb: "8px" }}
-                          >
+                            sx={{
+                              textAlign: isActive && "center",
```

Comment:
Not really part of this PR, but it would be good to remove `isActive` as possible to have the styles update with CSS media queries instead of re-renders.
```suggestion
                              textAlign: {xs: "center", sm: "left"},
```